assets/js/vue-apps/components/budget-input.vue

```diff
@@ -43,102 +42,93 @@ export default {
                             v-model="lineItem.cost"
                             min="1"
                             :max="maxBudget"
-                            class="ff-text ff-text--currency"
+                            class="ff-currency__input"
                         />
-                    </td>
-                    <td v-if="budgetRows.length > 1 && index !== budgetRows.length - 1">
-                        <button
-                            class="btn btn--small btn--outline u-block-full"
-                            type="button"
-                            @click="removeItem(lineItem)"
-                        >
+                    </div>
+                </div>
+                <div class="ff-budget-row__action">
+                    <button
+                        class="btn btn--small btn--outline"
+                        type="button"
+                        @click="removeItem(lineItem)"
+                        v-if="canDelete(index)"
+                    >
+                        <span class="btn__icon btn__icon-left">
                             <IconBin id="delete-icon" description="Delete this row" />
-                            Delete row
-                        </button>
-                    </td>
-                </tr>
-
-                <tr v-if="error" v-cloak>
-                    <td class="ff-budget__error">
-                        <!-- @TODO localise -->
-                        <span v-if="error.TOO_MANY_ITEMS">
-                            You have added the maximum number of budget rows available ({{ maxItems }}).
                         </span>
-                        <span v-if="error.OVER_BUDGET">
-                            You have exceeded the budget limit for this application of £{{
-                                maxBudget.toLocaleString()
-                            }}.
-                        </span>
-                    </td>
-                </tr>
-            </tbody>
+                        Delete row
+                    </button>
+                </div>
+            </li>
+        </ol>
+
+        <div class="ff-budget__errors" aria-live="polite" aria-atomic="true">
+            <!-- @TODO localise -->
+            <p v-if="error.TOO_MANY_ITEMS">
+                You have added the maximum number of budget rows available ({{ maxItems }}).
+            </p>
+            <p v-if="error.OVER_BUDGET">
+                You have exceeded the budget limit for this application of £{{ maxBudget.toLocaleString() }}.
+            </p>
+        </div>
 
-            <tfoot v-cloak>
-                <tr>
-                    <td><span class="ff-budget__label">Total:</span></td>
-                    <td>
-                        <span class="ff-budget__total" tabindex="0">£{{ total }}</span>
-                    </td>
-                    <td></td>
-                </tr>
-            </tfoot>
-        </table>
+        <div class="ff-budget__total" aria-live="polite" aria-atomic="true">
```

Comment:
`aria-atomic` pairs with `aria-live` to encourage screen readers to read out the whole text, default behaviour is to only read out the text that has changed,